packages/components/psammead-bulleted-list/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PsammeadBulletedList should render correctly from ltr 1`] = `
+.c0 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  list-style-type: none;
+}
+
+.c0 > li::before {
+  content: ' ';
+  display: inline-block;
+  width: 1.5rem;
+  background: url("function%20SvgBullet(props)%20%7B%0A%20%20return%20React.createElement(%0A%20%20%20%20'svg'%2C%20%0A%20%20%20%20Object.assign(%7B%7D%2C%20props%2C%20%7B'data-file-name'%3A%20SvgBullet.name%7D)%0A%20%20)%3B%0A%7D") no-repeat left center;
+  margin-left: -1.5rem;
+  background-size: 30%;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c0 {
+    font-size: 1rem;
+    line-height: 1.375rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    font-size: 1rem;
+    line-height: 1.375rem;
+  }
+}
+
+<ul
+  class="c0"
+  dir="ltr"
+>
+  <li>
+    First item on the list
+  </li>
+  <li>
+    Second item on the list
+  </li>
+  <li>
+    Final list item
+  </li>
+</ul>
+`;
+
+exports[`PsammeadBulletedList should render correctly from rtl 1`] = `
+<ul
```

Comment:
Does this look correct in RTL?  Weird that there are no styles in this snapshot, but there are styles in the above snapshot 🤔

Reply:
For some reason if you move all the css from `dirMargin` and `dirBackground` into `BulletedList`, this snapshot gets updated and all of the styles are added. Seems like there might have been a problem with passing/reading `dir` correctly.